crash_similarity.py

```diff
@@ -15,10 +15,23 @@
 import download_data
 import utils
 
+import datetime
 import pyximport
 pyximport.install()
 
 
+# Checks if the model has been trained in the last 24 hours (using datetime.timedelta class)
+def check_training_time(time_interval):
```

Comment:
This is still implemented in a strange way; why are you converting dates to timedeltas? You can subtract dates directly 😃 

The function should not have any argument. It should read the date from the file directly.